.circleci/config.yml

```diff
@@ -164,7 +164,7 @@ jobs:
       - run:
           name: Run e2e admin page test
           command: |
-            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="adminPage" --prod_env
+            python -m scripts.run_e2e_tests --skip-install --skip-build --suite="adminPage" --prod_env --deparallelize-terser
```

Comment:
The flag shouldn’t be needed here since the build is skipped.

Reply:
Done